_episodes/03-matplotlib.md

```diff
@@ -62,7 +75,7 @@ matplotlib.pyplot.show()
 
 ![Minimum Value Along The First Axis](../fig/inflammation-01-minimum.svg)
 
-The maximum value rises and falls smoothly, while the minimum seems to be a step function.  Neither
+The maximum value rises and falls smoothly, while the minimum values seems to be a step function.  Neither
```

Comment:
@ldko, could you please comment on this change?

Reply:
I don't think we need this change. Since this refers to the value per day across all patients, I don't think we need to add "values"--it also makes the sentence inconsistent with "maximum value" (singular) and a mismatch grammatically with "values" and "seems".